tests/components/emulated_hue/test_hue_api.py

```diff
@@ -255,7 +279,7 @@ async def test_light_without_brightness_can_be_turned_on(hass_hue, hue_client):
 
     no_brightness_result_json = await no_brightness_result.json()
 
-    assert no_brightness_result.status == 200
+    assert no_brightness_result.status == HTTP_OK
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5462445)